src/components/ActivityList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import ActivityCard from './ActivityCard'
+import styled from 'styled-components'
+
+const StyledActivityContainer = styled.section`
+  margin: 10px;
+`
+
+const StyledActivityListHeadline = styled.div`
```

Comment:
here same as above. maybe you can use a `header` element and name it `StyledActivityListHeader`. When you name it `Headline` I really would expect a headline but in fact you have two lines with `h2`and `p`

Reply:
I'll have to research the recommended use cases for header and change it :)